include/hex.h

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
I think we can fit this is in `util.h`, no need for a new header?
https://github.com/zephyrproject-rtos/zephyr/blob/master/include/misc/util.h

Reply:
Sounds good, I will do that.